src/modules/menu/components/settings/MoreInfo.vue

```diff
@@ -0,0 +1,31 @@
+<script setup>
+import { computed } from 'vue'
+import { useI18n } from 'vue-i18n'
+import { useStore } from 'vuex'
+
+import HeaderLink from '@/modules/menu/components/header/HeaderLink.vue'
+
+const store = useStore()
+const i18n = useI18n()
+
+const isDesktopMode = computed(() => store.getters.isDesktopMode)
+const cmsUrl = computed(() => i18n.t('cms_link_button_url'))
+
+function openCmsLink() {
+    window.open(cmsUrl.value, '_blank', 'noreferrer')
+}
+</script>
+
+<template>
+    <HeaderLink v-if="isDesktopMode" @click="openCmsLink">
+        {{ $t('cms_link_button_title') }}
+    </HeaderLink>
+    <button
```

Comment:
![image](https://github.com/geoadmin/web-mapviewer/assets/11540521/5b24c993-cc98-4fea-9c57-af9bad5a615d)

there's a little gap missing between the two "groups" of button (same as the one between the two button of the first line)
Maybe we should place this button inside the same "container" as the two other button, so that if there's enough space (depending on device and selected lang) they can be all three displayed on the same line